src/PageRouter.js

```diff
@@ -74,6 +75,14 @@ export function PageRouter() {
               </RequireAuth>
             }
           />
+          <Route
+            path="/createjob"
+            element={
+              <RequireAuth>
+                <CreateJobPage />
```

Comment:
I didn't think `CreateJob` was its own page. Wasn't starting a job just going to be done from an investor and a Modal ?